ambry-protocol/src/main/java/com.github.ambry.protocol/AmbryRequests.java

```diff
@@ -398,16 +396,11 @@ public void handleDeleteRequest(NetworkRequest request) throws IOException, Inte
         response = new DeleteResponse(deleteRequest.getCorrelationId(), deleteRequest.getClientId(), error);
       } else {
         BlobId convertedBlobId = (BlobId) convertedStoreKey;
-        MessageFormatInputStream stream =
-            new DeleteMessageFormatInputStream(convertedStoreKey, convertedBlobId.getAccountId(),
-                convertedBlobId.getContainerId(), deleteRequest.getDeletionTimeInMs());
-        MessageInfo info = new MessageInfo(convertedStoreKey, stream.getSize(), convertedBlobId.getAccountId(),
-            convertedBlobId.getContainerId(), deleteRequest.getDeletionTimeInMs());
-        ArrayList<MessageInfo> infoList = new ArrayList<MessageInfo>();
-        infoList.add(info);
-        MessageFormatWriteSet writeSet = new MessageFormatWriteSet(stream, infoList, false);
+        MessageInfo info =
+            new MessageInfo(convertedStoreKey, 0, convertedBlobId.getAccountId(), convertedBlobId.getContainerId(),
```

Comment:
-1 might be better, to indicate invalid or meaningless.

Reply:
sure